num_threads: 64
update_hz: 20
gp_sdf_area_scale: 4
offset_distance: 0.02
zero_gradient_threshold: 9.9999999999999995e-07
max_valid_gradient_var: 0.10000000000000001
invalid_position_var: 2
train_gp_immediately: false
gp_sdf:
  kernel:
    type: kMatern32
    alpha: 1
    scale: 1.2
    scale_mix: 1
    weights:
      []
  log_lambda: 10
test_query:
  max_test_valid_distance_var: 0.40000000000000002
  search_area_half_size: 4.7999999999999998
  use_nearest_only: false